### PR TITLE
Angular 10.1 update 

### PR DESCRIPTION
**Pull request by santam85**
Ran latest schematics, fixed lint issues

### CODE REVIEW

File: projects/ng2-charts/src/lib/base-chart.directive.ts [hunk 1]
```diff
@@ -409,7 +407,7 @@ export class BaseChartDirective implements OnChanges, OnInit, OnDestroy, DoCheck
 
     const mergedOptions = this.smartMerge(options, this.themeService.getColorschemesOptions());
 
-    const chartConfig: ChartConfiguration = {
+    return {
```
Comment (paviad) [inline projects/ng2-charts/src/lib/base-chart.directive.ts:410]: I did this for type safety, but ok :)

Comment (santam85) [reply projects/ng2-charts/src/lib/base-chart.directive.ts:410]: It's implicit in the function return type:
`getChartConfiguration(): ChartConfiguration`

**Comment by paviad** [comment]
try to see why tests dont pass...

**Comment by santam85** [comment]
It's Applitools detecting differences...

**Comment by paviad** [comment]
Can you make a fresh applitools workspace so we can get the tests to pass
again?

**Comment by santam85** [comment]
Will remove the dependency from Applitools altogether for visual testing then, and replace it with a screenshot-based cypress plugin.

**Pull request merged**